### PR TITLE
fix: remove_admin and update_owner, update_admin are removed

### PR DESCRIPTION
**Pull request by gcranju**
### Commit Message

```bash
fix: remove_admin and update_owner, update_admin are removed
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)
- [ ] I have added version bump label on PR.

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: contracts/cosmwasm-vm/cw-common/src/xcall_connection_msg.rs [hunk 1]
```diff
@@ -11,6 +11,9 @@ pub enum ExecuteMsg {
     SetAdmin {
         address: String,
     },
+    TransferAdmin {
+        address: String,
+    },
     SetXCallHost {
```
Comment (ibrizsabin) [inline contracts/cosmwasm-vm/cw-common/src/xcall_connection_msg.rs:17]: I dont think we need update/ transfer admin. set admin should be enough and should be called by either or owner. @AntonAndell  any thoughts?

**Review by ibrizsabin — COMMENTED** [review]
Take antons opinion on this as well

File: contracts/cosmwasm-vm/cw-common/src/xcall_connection_msg.rs (hunk 1 above)
Comment (sabinchitrakar) [reply contracts/cosmwasm-vm/cw-common/src/xcall_connection_msg.rs:17]: @gcranju  anton has confirmed that only set admin should be enough and it should be called only by admin . In init message we will have a initial admin address that can set another address as admin.

**Review by ibrizsabin — DISMISSED** [review]

**Review by ibrizsabin — APPROVED** [review]
Nice One!

**Pull request merged**